docker/addons/twins/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+# This docker-compose file contains optional opcua-adapter and opcua-redis services
+# for the Mainflux platform. Since this services are optional, this file is dependent on the
+# docker-compose.yml file from <project_root>/docker/. In order to run these services,
+# core services, as well as the network from the core composition, should be already running.
+
+version: "3.7"
+
+networks:
+  docker_mainflux-base-net:
+    external: true
+
+volumes:
+  mainflux-twins-db-volume:
+  mainflux-twins-db-configdb-volume:
+
+services:
+  twins-db:
+    image: mongo:3.6-jessie
```

Comment:
Use new Mongo release, test with it. Use Ubuntu, as they switched to Ubuntu (Bionic I think).

Reply:
Is there a particular reason for sticking with old MondoDB release?